### PR TITLE
[QC-383] Rerunning post-processing with selected timestamps

### PR DESCRIPTION
**Pull request by knopers8**

### CODE REVIEW

File: Framework/include/QualityControl/PostProcessingRunner.h [hunk 1]
```diff
@@ -49,6 +49,8 @@ class PostProcessingRunner
   void stop();
   /// \brief Reset transition. Throws on errors.
   void reset();
+  /// \brief Runs the task over selected timestamps, performing the full start, run, stop cycle.
```
Comment (Barthelemy) [inline Framework/include/QualityControl/PostProcessingRunner.h:52]: could you add a line about the expected argument ?

**Review by Barthelemy — COMMENTED** [review]
just a small comment, otherwise, all good.

File: Framework/include/QualityControl/PostProcessingRunner.h (hunk 1 above)
Comment (knopers8) [reply Framework/include/QualityControl/PostProcessingRunner.h:52]: done

**Comment by knopers8** [comment]
testCcdbDatabase is failing randomly again https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/QualityControl/462/4242a7342aab0dbe355ac2bb9bae4f53edf79c75/build_QualityControl_o2-dataflow/fullLog.txt

**Comment by Barthelemy** [comment]
unfortunately the logs are ok now

**Review by Barthelemy — APPROVED** [review]

File: Framework/include/QualityControl/PostProcessingRunner.h (hunk 1 above)
Comment (Barthelemy) [reply Framework/include/QualityControl/PostProcessingRunner.h:52]: ty

**Comment by knopers8** [comment]
Meh, i thought that each build has a different link, so saving it would be enough.

**Pull request merged**